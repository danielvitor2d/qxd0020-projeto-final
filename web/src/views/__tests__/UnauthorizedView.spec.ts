import { shallowMount } from '@vue/test-utils';
import { describe, expect, it } from 'vitest';

import UnauthorizedViewVue from '../UnauthorizedView.vue';

describe('Unauthorized', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(UnauthorizedViewVue);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
