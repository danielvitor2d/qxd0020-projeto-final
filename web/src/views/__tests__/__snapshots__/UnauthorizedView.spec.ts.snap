// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Unauthorized > renders correctly 1`] = `
"<div class=\\"bg-[#FEFEFE] min-h-screen flex flex-col items-center justify-center\\">
  <div class=\\"text-black text-4xl mb-6\\">
    <font-awesome-icon icon=\\"fa-lock\\"></font-awesome-icon> Acesso não autorizado
  </div>
  <div class=\\"text-black text-lg text-center\\"> Desculpe, mas você não está autorizado a acessar esta página. Entre em contato com o administrador para obter mais assistência. </div>
  <div class=\\"text-black text-lg text-center\\"> Retornar para a <b>
      <router-link to=\\"/\\">Página Inicial</router-link>
    </b></div>
</div>"
`;
